src/interceptors/response.ts

```diff
@@ -87,7 +87,10 @@ export class CacheResponseInterceptor<R, D>
 
     let ttl = response.config.cache?.ttl || this.axios.defaults.cache.ttl;
 
-    if (response.config.cache?.interpretHeader) {
+    if (
+      response.config.cache?.interpretHeader ||
+      this.axios.defaults.cache.interpretHeader
```

Comment:
You're adding the feature to use default axios configuration, this should show in the commit description, to end up in the changelog :)

Reply:
I wouldn't call it a feature, as I had to be using the default configuration all along.